### PR TITLE
[DROOLS-4259] Scenario can not access parent data object fields

### PR DESCRIPTION
**Pull request by danielezonca**
@jomarko @gitgabrio @yesamer 

https://issues.jboss.org/browse/DROOLS-4259

### CODE REVIEW

File: drools-scenario-simulation/drools-scenario-simulation-backend/src/test/java/org/drools/scenariosimulation/backend/util/ScenarioBeanUtilTest.java [hunk 1]
```diff
@@ -197,4 +200,13 @@ public void loadClassTest() {
                 .isInstanceOf(ScenarioException.class)
                 .hasMessage("Impossible to load class NotExistingClass");
     }
+
+    @Test
+    public void getFieldTest() {
+        assertNotNull(getField(Person.class, "firstName"));
+        assertNotNull(getField(SubPerson.class, "firstName"));
```
Comment (jomarko) [inline drools-scenario-simulation/drools-scenario-simulation-backend/src/test/java/org/drools/scenariosimulation/backend/util/ScenarioBeanUtilTest.java:207]: Could we please add an assertion `assertNotNull(getField(SubPerson.class, "additionalField"));`

**Review by jomarko — COMMENTED** [review]
Just one comment, otherwise looks and works good.

File: drools-scenario-simulation/drools-scenario-simulation-backend/src/test/java/org/drools/scenariosimulation/backend/util/ScenarioBeanUtilTest.java (hunk 1 above)
Comment (danielezonca) [reply drools-scenario-simulation/drools-scenario-simulation-backend/src/test/java/org/drools/scenariosimulation/backend/util/ScenarioBeanUtilTest.java:207]: Done :)

**Review by jomarko — APPROVED** [review]
@danielezonca thank you.

**Review by gitgabrio — APPROVED** [review]
@danielezonca

**Pull request merged**